pkgs/servers/gotify/mod-sha.nix

```diff
@@ -1 +0,0 @@
-"0zpdbj8a0akhzi1ian8zs6y7ymhlpcfy13q64xdlwc5w0286554r"
```

Comment:
Removing this file will break [the update script](https://github.com/NixOS/nixpkgs/blob/b59b7414c35e5c4026c53453329a25e4260b958b/pkgs/servers/gotify/update.sh) (which also requires other adaptations for this change).

Reply:
cc @doronbehar

Reply:
:+1:. See https://github.com/NixOS/nixpkgs/pull/87849